packages/eslint-plugin-gestalt/src/__fixtures__/prefer-box-lonely-ref/invalid/gestalt-import-with-box-input.js

```diff
@@ -0,0 +1,10 @@
+import { Box } from 'gestalt';
+export default function TestElement() {
+  return (
+    <Box>
+      <div ref={undefined}>
```

Comment:
Presumably this works fine, but should we have at least one test where the ref actually has a value?

Reply:
DONE for all tests